Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Python runtime as a parent image
```

Comment:
I'm using Python 3.12, can we please update the Python version in this Dockerfile?

Reply:
@Aiderbot I'm using Python 3.12, can we please update the Python version in this Dockerfile?